usr/src/cmd/ptools/pwait/pwait.c

```diff
@@ -98,7 +100,7 @@ main(int argc, char **argv)
 		(void) enable_extended_FILE_stdio(-1, -1);
 	}
 
-	pollfd = (struct pollfd *)malloc(argc*sizeof (struct pollfd));
+	pollfd = (struct pollfd *)malloc(argc * sizeof (struct pollfd));
```

Comment:
maybe calloc here -- it'll also handle overflow of the two args.

Reply:
Also, I think the cast is probably superfluous

Reply:
> This looks like it needs to be upstreamed as well, no?

Well, the general change to `strlcat()`  is good, but the problem is only in smartos/omnios because of lx zones and the possible `/native/proc` prefix.